src/contexts/ThemeContext.tsx

```diff
@@ -0,0 +1,46 @@
+import { createContext, useState } from "react";
+import { ThemeProvider } from "styled-components";
+import { lightTheme, darkTheme } from "../themes/ThemeConfig";
+
+type MyThemeContextType = {
+  theme: string;
+  setTheme: (newTheme: string) => void;
+};
+export const MyThemeContext = createContext({} as MyThemeContextType);
+
+interface MyThemeProviderProps {
+  //disponibliza o theme e o setTheme para meu context
```

Comment:
Could you translate your comment into English? It is better for other colleagues to help you.